manifests/run.pp

```diff
@@ -218,7 +218,10 @@
         ensure    => $running,
         enable    => true,
         hasstatus => $hasstatus,
-        require   => File[$initscript],
+        require   => [
+          File[$initscript],
+          Service[$service_name]
+        ]
```

Comment:
Didn't want to edit this as part of this branch/PR, but it was the only way to get setup / docker run to work since "docker run" wasn't waiting for the service to be fully configured before attempting to run.